tests/functional/cylc-cat-log/03-bad-workflow.t

```diff
@@ -20,7 +20,7 @@
 set_test_number 4
 
 CYLC_RUN_DIR="$RUN_DIR"
-BAD_NAME="$(basename "$(mktemp -u "${CYLC_RUN_DIR}/XXXXXXXX")")"
+BAD_NAME="NONEXISTENTWORKFLOWNAME"
```

Comment:
(Unrelated: this test was flaky - mktemp can generate strings starting with digits, resulting in "invalid workflow name" errors instead of the intended ones).